[CI] DebMetadataTests test05CheckLintian failing

**Build Scans:**
- [elasticsearch-periodic-packaging #11541 / ubuntu-2204_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/fwtdcpiawbn66)
- [elasticsearch-pull-request #122101 / ubuntu-2404_destructivedistrotest.packages_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/iahpr2vsnu3vi)
- [elasticsearch-pull-request #121771 / ubuntu-2404_destructivedistrotest.packages_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/j52bf2nqdw5ms)

**Reproduction Line:**
```
null
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.packaging.test.DebMetadataTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test05CheckLintian'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Issues for DEB package found by Lintian:
Issue[severity=WARNING, tag=codeless-jar, message=usr/share/elasticsearch/modules/esql-datasource-grpc/grpc-context-1.78.0.jar]
```

**Issue Reasons:**
- [main] 3 failures in test test05CheckLintian (0.8% fail rate in 355 executions)
- [main] 2 failures in step ubuntu-2404_destructivedistrotest.packages_packaging-tests-unix-sample (0.6% fail rate in 337 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.6% fail rate in 337 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -407,9 +407,6 @@ tests:
 - class: org.elasticsearch.action.search.SearchContextInvalidationIT
   method: testScrollReturns404WhenNodeLeavesCluster
   issue: https://github.com/elastic/elasticsearch/issues/142810
-- class: org.elasticsearch.packaging.test.DebMetadataTests
-  method: test05CheckLintian
-  issue: https://github.com/elastic/elasticsearch/issues/142819
 - class: org.elasticsearch.xpack.search.AsyncSearchTaskTests
   method: testWithFailure
   issue: https://github.com/elastic/elasticsearch/issues/142821

